plugins/janus_videoroom.c

```diff
@@ -554,7 +554,7 @@ room-<unique room ID>: {
  * the response to a \c joinandconfigure will be a \c joined event, and
  * will again be accompanied by a JSEP SDP answer as usual.
  *
- * However you decided to publish something, as soon as the PeerConnection
+ * However if you decided to publish something, as soon as the PeerConnection
```

Comment:
The "if" is wrong here: what I meant was "no matter how you decided to publish something" (since we just talked of different ways of doing that).